src/cfg.cpp

```diff
@@ -203,7 +203,7 @@ void read_cfg()
 	show_version = config["show_version"].value_or(true);
 	window_size_x = config["window_size_x"].value_or(0);
 	window_size_y = config["window_size_y"].value_or(0);
-	internal_resolution_scale = config["internal_resolution_scale"].value_or(2);
+	internal_resolution_scale = config["internal_resolution_scale"].value_or(0);
```

Comment:
If you update this value, you should updated it also here: https://github.com/julianxhokaxhiu/FFNx/blob/master/misc/FFNx.toml#L43

Reply:
Ah ok, I wondered if the FFNx.toml was in there somewhere as well.  I'll make the change there as well.